pkg/activator/net/lb_policy.go

```diff
@@ -84,3 +85,34 @@ func firstAvailableLBPolicy(ctx context.Context, targets []*podTracker) (func(),
 	}
 	return noop, nil
 }
+
+type roundRobinPolicyT struct {
+	mu  sync.Mutex
+	idx int
+}
```

Comment:
The extra type seems unncecessary :thinking:

Reply:
sure I can inline it.